organizations.json

```diff
@@ -667,9 +678,9 @@
         "city": "Accra, Ghana",
         "latitude": "5.5811",
         "longitude": "-0.1990",
-        "type": "Code for All",
         "tags": [
-            "Code for All"
+            "Code for All",
+            "Affiliate"
```

Comment:
Could you update the "tags" section in the README with descriptions of what "Affiliate" and "Fellowship" mean.